install.md

```diff
@@ -33,9 +33,7 @@ gcloud projects list
 
 ## Step #2: Enable the requisite API's for your Google Cloud Project
 
-NOTE: Your account must be whitelisted to enable the Container Analysis API. To do so, join the  [Container Analysis Users Group](https://groups.google.com/forum/#!forum/containeranalysis-users). It may take 1-5 business days to approve the request.
```

Comment:
Thank you! Great! you able to to use account without the whitelist?

Reply:
just read your comment in other PR. Looks like you were able too. Thanks for deleting this!